Tasks/Gradle/README.md

```diff
@@ -0,0 +1,69 @@
+#Build your code using Gradle in VSTS
+
+###Paramaters for Gradle build task are explained below
+
+- **Gradle Wrapper :** This is a Required field. The location in the repository of the gradlew wrapper used for the build. Note that on Windows build agents (including the hosted pool), you must use the `gradlew.bat` wrapper. Xplat build agents use the `gradlew` shell script. To Know more [click here](https://docs.gradle.org/current/userguide/gradle_wrapper.html)
+
+- **Options :** Specify any command line options you want to pass to the Gradle wrapper. To know more [click here](https://docs.gradle.org/current/userguide/gradle_command_line.html)
+
+- **Goal(s) :** The task(s) for Gradle to execute. A list of tasks can be taken from `gradlew tasks` issued from a command prompt. To knoq more [click here](https://docs.gradle.org/current/userguide/tutorial_using_tasks.html)
+
+####JUnit Test Results
+Use the next three options to manage your JUnit test results in VSTS
+
+- **Publish to VSTS :** Select this option to publish JUnit Test results produced by the Gradle build to Visual Studio Team Services/TFS. Each test result file matching `Test Results Files` will be published as a test run in VSTS.
+
+- **Test Results Files :** This option wil appear if you select the above option. Here, provide Test results files path. Wildcards can be used. For example, `**/TEST-*.xml` for all xml files whose name starts with `TEST-."`
+
+- **Test Run Title :** This option will appear if you select the `Publish to VSO/TFS` option. Here provide a name for the Test Run
+
+####Code Coverage
+Use the next options to manage your code coverage options. If your Build file already has Code Coverage enabled, you can ignore this section and use the Publish Code Coverage task to upload results to TFS/VSTS
+
+- **CodeCoverage Tool :** Select the code coverage tool you want to use. Currently JaCoCo and Cobertura are supported. For JaCoCo, make sure `jacocoant.jar` is available in lib folder of Ant installation. For cobertura, set up an environment variable `COBERTURA_HOME` pointing to the cobertura jars location. [Click here](http://www.eclemma.org/jacoco/trunk/doc/ant.html) to know more about Jacoco and [Click here](https://github.com/cobertura/cobertura/wiki/Ant-Task-Reference) to know more about Cobertura.
```

Comment:
Updated the files with suggested changes. I could'nt find proper links for Gradle with Cobertura, hence removed the link references in Gradle read me file.